templates/web/pages/session/signed_in.tt.html

```diff
@@ -4,7 +4,9 @@
 
 [% IF !server_options_producers_platform %]
     <h3>[% lang('you_can_also_help_us') %]</h3>
```

Comment:
Thanks for the PR @jnsereko!

Could you include this <h3> title inside the IF below? Otherwise we show "Help us to fund the OFF project" but with nothing under it.

Reply:
Hey @stephanegigandet 
i have added the `<h3>` into the if, but i didn't like below but i didn't understand pretty well what you meant with "Otherwise we show "Help us to fund the OFF project" but with nothing under it."
```
    [% IF lc == "en" or lc == "fr" %]
        <h3>[% lang('you_can_also_help_us') %]</h3>
        <p>[% lang('bottom_content') %]</p>
    [% END %]
```

Reply:
I meant that the h3 tag goes with the p tag, if we don't display the p tag, then we don't want to display just a h3 title with nothing under it.